libraries/SITL/SIM_Precland.cpp

```diff
@@ -105,6 +105,13 @@ const AP_Param::GroupInfo SIM_Precland::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("ORIENT", 9, SIM_Precland, _orient, ROTATION_PITCH_90),
 
+    // @Param: DST_EN
```

Comment:
This could be an options parameter instead of just a distance-enable parameter.

Reply:
Done! :)